package.json

```diff
@@ -53,7 +53,7 @@
     "lint:all": "yarn lint:core && yarn lint:clippy",
     "lint": "eslint --ext js,jsx --quiet",
     "clean": "lerna clean",
-    "build": "lerna run build --stream --parallel",
+    "build": "lerna run build --stream",
```

Comment:
These had to be made non-parallel so that core can access items from `dist` inside the components package

Reply:
Yup, this can be parallel with the entrypoint set to the non-compiled code